samples/tiny_http/src/main.rs

```diff
@@ -54,5 +67,15 @@ fn process_entries(entries: Entries) -> Result<(), Error> {
         println!(r#"Field "{}" is file "{}":"#, name, filename);
         println!("{}", contents);
     }
-    Ok(())
+    Ok(build_response(200, "Multipart data is received!".into()))
```

Comment:
@cybergeek94 yes, completely forgot. Strange that neither rustc, nor clippy did warn about this.